requirements-dev.txt

```diff
@@ -3,6 +3,8 @@ pre-commit
 pytest==7.1.3
 pytest-asyncio==0.19.0
 pytest-timeout==2.1.0 # used to timeout tests
+pytest-xdist==2.5.0  # used to run tests in parallel
+pytest-xdist[psutil]  # used for better auto detection of cpu cores to run in parallel
```

Comment:
```
pytest-xdist[psutil]==2.5.0
```
doesn't that work?

Reply:
hm, didn't think chaining these would work, but it seems that it does